lib/addDefaults.js

```diff
@@ -10,6 +10,53 @@ var findAccessorMinMax = require('./findAccessorMinMax');
 
 module.exports = addDefaults;
 
+/**
+ * Adds default values to any missing glTF values of the hierarchy.
+ *
+ * @param {Object} gltf - An object holding a glTF hierarchy
```

Comment:
For the sake of consistency with other areas in this project, let's just remove the hyphens.